projects/observability/src/shared/components/topology/d3/interactions/zoom/topology-zoom.ts

```diff
@@ -258,7 +258,7 @@ export class TopologyZoom<TContainer extends Element = Element, TTarget extends
 
   private isPrimaryMouseOrTouchEvent(receivedEvent: ZoomSourceEvent): receivedEvent is TouchEvent | MouseEvent {
     return (
-      receivedEvent instanceof TouchEvent ||
+      receivedEvent instanceof window.TouchEvent ||
```

Comment:
Does FF have another global TouchEvent? Why does this need qualification?

Reply:
This interface is not defined by FF. The qualification avoids the reference error

Reply:
I have seen Firefox complain many times that TouchEvent is not defined. And, that used to come on topology page